posts/2015-06-12-best-of-web-2015.md

```diff
@@ -0,0 +1,58 @@
+Il nous a donc fait un petit récap de la situation actuelle et des solutions pour contourner le problème de faire des grilles en css (Bootstrap, 960 grid, Foundation, Flexbox) mais de la réalité de celles-ci qui ne sont pas encore optimale. On a pu ensuite assister à une démonstration de comment CSS Grid va régler tout cela (une fois supporté) grace à une syntaxe très simple sans avoir à faire appel à des librairies. 
+
+## La [« Winning Stack » et l’ES6]( http://fr.slideshare.net/SfeirGroup/es2015-ready-angular-web-stack-bestofweb-2015) par Douglas Duteil
+Douglas Duteil nous a présenté sa winning stack avec angular et ES6. Il nous a parlé de systemJs pour la chargement dynamic de module, JSPM pour la gestion des packages, de quelques différences de ES6 (2015) et ES5 (2009), de babel, Istanbul ainsi que de [Isparta]( https://github.com/douglasduteil/isparta) (créé et maintenu par Douglas). D’ailleurs, si le projet Isparta vous intéresse, il cherche de l’aide pour le merger à Istanbul
+
+## [Web Audio API](http://ouhouhsami.github.io/2015-06-05-bestofweb-paris/) par Samuel Goldszmidt et Norbert Schnell de l'IRCAM
+Samuel Goldszmidt et Norbert Schnell sont deux chercheurs et développeurs de l'[IRCAM](http://www.ircam.fr/). Ils nous ont fait découvrir les travaux actuellement en cours à l'IRCAM par rapport à l'utilisation des technologies Web et plus précisement la Web Audio API. Leur présentation était intéractive, ils ont commencé par nous montrer quelque travaux réalisés avec la Web Audio API(Amplificateur, filtre, édition de données temporelles, traitement audio). Ils nous ont ensuite présenté le Collective-Soundworks, un framework pour la création d'application audio et multimédia de manière collaborative auquel ils participent et nous on fait une démonstration collaborative en nous demandant de nous connecter à une url pour que l'ensemble de nos terminaux jouent ensemble certaines sonorités.
+Une présentation assez captivante et qui donne beaucoup d'idées d'applications.
+
+## Le [Material design avec Polymer](https://docs.google.com/presentation/d/1IzsxsE6HybPAdbrI8iIlF3Qaj9BQGEHmu3zX7xT9o8M/edit#slide=id.g3a6159c6f_024) par Martin Gorner
+Martin Gorner a commencé sa présentation en nous rappellant ce qui constitue le Material Design, ces éléments (paper, touch first, couleur textuelle, animation, bouton flottant, ...). Il nous a ensuite fait une démonstration de Polymer (qui vient de sortir en version 1.0), quelques petits bower install plustard et c'est parti, on charge polymer dans la balise script et on peut utiliser les paper-card en utilisant des ``<link rel="import" href`` pour les importer et ensuite directement dans notre code html. La bibliothéque de base de Polymer est déjà bien rempli mais il est également possible de créer ses propres cards et de bien s'amuser avec le Material Design. Polymer donne vraiment envie, pouvoir créer une interface Material Design en quelque coups d'import, ça promet de belles choses.
```

Comment:
:+1:  for the use of the word _bibliothèque_ instead of _librairie_ as often seen in french context.